app/views/cooks/index.html.erb

```diff
@@ -23,6 +23,9 @@
       <td><%=cook.name%></td>
       <td><%=cook.cuisine%></td>
       <td><%=cook.city%></td>
+      <td><%= link_to 'Show', cook %></td>
+      <td><%= link_to 'Edit', edit_cook_path(cook) %></td>
+      <td><%= button_to 'Destroy', cook, method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
Do you have a tests for these three actions?

Reply:
Do you want the confirmation dialog to appear?

Reply:
You're right, my next piece of work is to write tests for the Edit and Destroy paths. This PR is concerned with the Show.

Reply:
I like the confirmation dialog. Let me know your thoughts on this.

Reply:
There is a school of thought that says you shouldn't ask for confirmation, but you should instead offer the visitor a way to undo their action. This is outside of the scope of what I would expect you to do, but you could mark a cook as deleted instead of actually deleting it. You could then show a flash on the index page after the "deletion" to allow the visitor to undo. You could also offer some functionality on the show page to allow visitors to undo their deletions. On top of all this, you could also have some filtering on the index page to show deleted cooks too. So many avenues!!!